src/cobra/io/web/embl_gems_repository.py

```diff
@@ -0,0 +1,98 @@
+"""
+Provide a concrete implementation of the carveme repository interface.
+"""
+
+
+from io import BytesIO
+
+import httpx
+
+from .abstract_model_repository import AbstractModelRepository
+
+
+def _decode_model_path(model_path):
+    """Decode the model path to EMBL GEMs."""
+    tokens = model_path.split("_")
+    genus = tokens[0]
+
+    directory = genus.lower()
+    alphabet = directory[0]
+
+    return f"{alphabet}/{directory}/{model_path}"
+
+
+class EMBLGems(AbstractModelRepository):
+    """
+    Define a concrete implementation of the EMBL GEMs repository.
+
+    Attributes
+    ----------
+    name : str
+        The name of the EMBL GEMs repository.
+
+    """
+
+    name: str = "EMBL GEMs"
+
+    def __init__(
+        self,
+        **kwargs,
+    ) -> None:
+        """
+        Initialize a EMBL GEMs repository interface.
+
+        Other Parameters
+        ----------------
+        kwargs
+            Passed to the parent constructor in order to enable multiple inheritance.
+
+        """
+        super().__init__(
+            url="https://github.com/cdanielmachado/embl_gems/blob/master/models/",
```

Comment:
Access the raw content directly:

```suggestion
            url="https://raw.githubusercontent.com/cdanielmachado/embl_gems/master/models/",
```